src/Domain/Stager.php

```diff
@@ -84,12 +108,28 @@ private function validatePreconditions(): void
         }
     }
 
-    private function runCommand(): void
+    /**
+     * @throws \Symfony\Component\Process\Exception\RuntimeException
```

Comment:
Looks like this actually throws `ProcessFailedException`.

Reply:
Actually not, because I'm catching `ProcessFailedException` in the method body. However, It doesn't look like anything throws `RuntimeException`, either. So I'm not sure how that got in there or why PHPStan didn't detect it! 😬 I'll remove it.